built_value_generator/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ environment:
   sdk: '>=2.0.0-dev <3.0.0'
 
 dependencies:
-  analyzer: ^0.33.1
+  analyzer: ^0.33.3
```

Comment:
@scheglov - Is the analyzer team prepared to support this API throughout the `0.33.x` version of Analyzer?

Reply:
@stereotype441 I know that there is a branch for analyzer `0.33.x` somewhere, but I was not able to find it on GitHub. IIRC we put this temporary non-API `ParsedLibraryResultImpl` and `ResolvedLibraryResultImpl` in this branch.

Reply:
@natebosch @scheglov Good point. I should pin this to exactly 0.33.3, I think?

Reply:
Yes, we'll continue supporting this backported API for the rest of `0.33.x`, so it shouldn't be necessary to pin to exactly 0.33.3.  In fact, it would be better not to pin, because if we discover the need to do additional backporting to support other clients, we'll have to release `0.33.4`, and if/when we do that, we don't want to have to modify built_value to allow it.

@scheglov The branch is called "analyzer-0.33".  It's not mirrored to github, but it exists in googlesource.com, so assuming you clone from https://dart.googlesource.com/sdk.git, you should see the branch there.

Reply:
Sounds good, thanks. I'll aim to land this tomorrow.